.goreleaser.yml

```diff
@@ -42,9 +50,11 @@ changelog:
 dockers:
   - dockerfile: Dockerfile.release
     use: buildx
+    goos: linux
+    goarch: amd64
     image_templates:
-      - "docker.io/aquasec/postee:{{ .Version }}"
-      - "public.ecr.aws/aquasecurity/postee:{{ .Version }}"
+      - "docker.io/aquasec/postee:{{ .Version }}-amd64"
+      - "public.ecr.aws/aquasecurity/postee:{{ .Version }}-amd64"
```

Comment:
Will we have to update the Tracee documentation/k8s config as well? I recall that it referenced this image too.

Reply:
the tracee documentation is using the yaml directly from Postee.
https://aquasecurity.github.io/tracee/v0.7.0/install/kubernetes/
```
kubectl create \
  -f https://raw.githubusercontent.com/aquasecurity/postee/main/deploy/kubernetes/postee.yaml \
  -f https://raw.githubusercontent.com/aquasecurity/tracee/v0.7.0/deploy/kubernetes/tracee-postee/tracee.yaml

```

Reply:
Yes but today we just pull the `postee:latest` image, without specifying the arch. I believe we will need to change that? https://github.com/aquasecurity/postee/blob/main/deploy/kubernetes/postee.yaml#L179

Reply:
This part of the latest will generated in the same way. https://github.com/aquasecurity/postee/blob/main/.goreleaser.yml#L48